### DISCUSSION TITLE
Way to fitView with default zoom?

### DISCUSSION THREAD

**Discussion by prksean** [bcakmakoglu/vue-flow#922] [Q&A] [answered]
Is there a way to fitView and set the desired zoom instead of having to show all nodes?

At first load, I want to set the view centered around my base node with specific zoom level regardless of how many nodes exist.

**Comment by bcakmakoglu** [comment, marked as answer]
You can pass a number of options to the `fitView` action.

```ts
export interface TransitionOptions {
  duration?: number
}

export type FitViewParams = {
  padding?: number
  includeHiddenNodes?: boolean
  minZoom?: number
  maxZoom?: number
  offset?: {
    x?: number
    y?: number
  }
  nodes?: string[]
} & TransitionOptions
```

1. setting `minZoom` and `maxZoom` to the same value will force a specific zoom level
2. setting id(s) to the `nodes` option will force fit view to only wrap around the specified node(s)
